File: HA5_5.py
#Write a program which accepts N and print first 5 Multiples of it
#Input : 4
#Output : 4    8    12     16      20

def Pattern(iNo):
    iCnt = 0
    for iCnt in range(1,5+1,1):
        print(iCnt*iNo)

def main():
    print("Enter a Number")
    iValue = int(input())
    
    Pattern(iValue)

if __name__ =="__main__":
    main()

File: HA5_5.cpp
//Write a program which accepts N and print first 5 Multiples of it
//Input : 4
//Output : 4    8    12     16      20

#include<iostream>
using namespace std;

void Pattern(int iNo)
{
    int iCnt = 0;
    for(iCnt =1; iCnt <= 5; iCnt++)
    {
        cout<<iNo*iCnt<<"\t";
    }
}
int main()
{
    int iValue = 0;
    cout<<"Enter a Number"<<"\n";
    cin>>iValue;

    Pattern(iValue);

    return 0;
}